ANSWERS.md

```diff
@@ -0,0 +1,18 @@
+q.  What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+a.  PropTypes are used for  checking data in JS to insure all required data is received.
+
+
+q. Describe a life-cycle event in React?
+
+a.  A life-cycle event changes the DOM making changes to the app prior to it being displayed to the user.
+
+q. Explain the details of a Higher Order Component?
+
+a. A HOC is a function that takes in a function or method and may return a function.
```

Comment:
That's the basic principle, although higher order components will return components